themes/src/main/resources/theme/keycloak/login/resources/css/tile.css

```diff
@@ -188,6 +192,7 @@
 
 .pf-c-tile__title {
     color: var(--pf-c-tile__title--Color);
+    overflow: hidden;
```

Comment:
@bodograumann Just one minor thing. Could you please add the `overflow` property directly to `.pf-c-tile` class? 
```
.pf-c-tile{
   ...
   overflow:hidden
}
```
For testing purposes we use test Keycloak Server [(Docs)](https://github.com/keycloak/keycloak/blob/master/docs/tests.md#live-edit-of-html-and-styles) with hot-reload of CSS and HTML, where you can test it. Even test the mobile versions basically through the browser JS console.

Reply:
Changing that is easy I guess. Why do you think this is better?

I actually have a testing setup for themes with docker and a volume, but getting back into that after a month requires some effort.

Reply:
@bodograumann Width of an element with class `pf-c-tile__title` is changing regarding the input length, so there's never overflow for the element. In parent element with class `pf-c-tile` we can manage the text overflow inside. I know it's not an ideal approach as well, but much better than the context overflow. You can compare the figures below with and without the overflow. WDYT? 

![with](https://user-images.githubusercontent.com/38039883/112831181-dc90c700-9093-11eb-864b-42c0b7148b40.png)
![without](https://user-images.githubusercontent.com/38039883/112831185-dd295d80-9093-11eb-95a7-7447216700b4.png)

Reply:
Oh I see. Thanks for testing and explaining @mabartos. I'll do the change right away.